client/my-sites/customer-home/cards/education/educational-content/index.jsx

```diff
@@ -25,7 +25,7 @@ const EducationalContent = ( { title, description, links, illustration } ) => {
 				</p>
 				<div className="educational-content__links">
 					{ links.map( ( { postId, url, text, icon, tracksEvent, statsName } ) => (
-						<div className="educational-content__link">
+						<div className="educational-content__link" key={ postId }>
```

Comment:
<img width="1405" alt="Screen Shot 2020-04-27 at 2 12 33 PM" src="https://user-images.githubusercontent.com/1270189/80421328-2758fc00-8891-11ea-8bd4-aa4c915b801a.png">

Mastering the gutenberg card has links to the same postId, @kwight do you mind following up with a more unique key, can maybe just add the anchor, or add a short rand number at the end.

Reply:
https://github.com/Automattic/wp-calypso/pull/41533

Reply:
oh dang – thanks for the follow up @mmtr 👍